LibreNMS/Alert/Transport/Api.php

```diff
@@ -50,7 +50,9 @@ private function contactAPI($obj, $api, $options, $method, $auth)
         //get each line of key-values and process the variables;
         foreach (preg_split("/\\r\\n|\\r|\\n/", $options) as $current_line) {
             list($u_key, $u_val) = explode('=', $current_line, 2);
-
+            if (empty($u_key)) {
```

Comment:
Probably would be nicer to just do this:
`preg_split("/\\r\\n|\\r|\\n/", '', -1, PREG_SPLIT_NO_EMPTY)`

Reply:
I cannot promise I'll remember that one, but I know where to search at least :) Thanx !